src/appleseed/renderer/modeling/light/sunlight.cpp

```diff
@@ -127,22 +126,20 @@ namespace
             // Evaluate uniform inputs.
             m_inputs.evaluate_uniforms(&m_values);
 
-
             Source* distance_src = get_inputs().source("distance");
             assert(distance_src != nullptr);
             if (distance_src->is_uniform())
                 distance_src->evaluate_uniform(m_values.m_distance);
             else
             {
                 RENDERER_LOG_WARNING(
-                    "Distance between Sun and scene \"%s\" is not uniform.",
+                    "distance between sun and scene \"%s\" is not uniform.",
```

Comment:
I can't understand

> Here we should probably use the default Sun-scene distance of 149.6 million km.

franz said.

Reply:
What I meant is this:

Currently, if the distance source is not uniform, you print a warning (which is the right thing to do) but `m_values.m_distance` is left uninitialized. I suggest to do two things:

1. In the case where the distance source is not uniform, use the default distance.
2. Modify the message to indicate that appleseed will use the default distance.

E.g.

    Source* distance_src = get_inputs().source("distance");
    assert(distance_src != nullptr);
    if (distance_src->is_uniform())
        distance_src->evaluate_uniform(m_values.m_distance);
    else
    {
        RENDERER_LOG_WARNING(
            "distance between sun and scene \"%s\" is not uniform, using default value of 149.6 million km.",
            get_path().c_str());
        m_values.m_distance = 149.6;
    }